### PR TITLE
[57632] Use FeedbackDialog component in core

### PR DESCRIPTION
**Pull request by HDinger**
# Ticket
https://community.openproject.org/projects/openproject/work_packages/57632/activity

# What are you trying to accomplish?
Bump primer to 0.44.3 and introduce FeedbackDialog and FeedbackMessage in the core

- [x] API Token generation
- [x] Storages setup success
- [x] Nudge dialog for user to login to Storage
- [x] Confirm removal of project from storage in "Enable in Projects"
- [x] Job status modal

# Merge checklist

- [x] Added/updated tests

### CODE REVIEW

File: modules/storages/spec/requests/project_storages_open_spec.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@
...
+                expect(last_response.body).to eq("<turbo-stream action=\"update\" target=\"open-project-storage-modal-body-component\">\n    <template>\n        \n  <div class=\"blankslate-container\">\n    <div data-view-component=\"true\" class=\"FeedbackMessage blankslate\">\n      <svg aria-hidden=\"true\" height=\"24\" viewBox=\"0 0 24 24\" version=\"1.1\" width=\"24\" data-view-component=\"true\" class=\"octicon octicon-check-circle blankslate-icon color-fg-success\">\n    <path d=\"M17.28 9.28a.75.75 0 0 0-1.06-1.06l-5.97 5.97-2.47-2.47a.75.75 0 0 0-1.06 1.06l3 3a.75.75 0 0 0 1.06 0l6.5-6.5Z\"></path><path d=\"M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1ZM2.5 12a9.5 9.5 0 0 0 9.5 9.5 9.5 9.5 0 0 0 9.5-9.5A9.5 9.5 0 0 0 12 2.5 9.5 9.5 0 0 0 2.5 12Z\"></path>\n</svg>\n\n      <h2 data-view-component=\"true\" class=\"blankslate-heading\">Integration setup completed</h2>\n      <p id=\"waiting_subtitle\" data-view-component=\"true\">You are being redirected</p>\n\n</div>  </div>\n\n\n\n    </template>\n</turbo-stream>\n\n") # rubocop:disable Layout/LineLength
```
Comment (oliverguenther) [inline modules/storages/spec/requests/project_storages_open_spec.rb:78]: I don't get this spec, why are why testing the exact DOM @opf/titan ?

**Review by oliverguenther — APPROVED** [review]

File: modules/storages/app/components/storages/open_project_storage_modal_component/body.html.erb [hunk 2]
```diff
@@ -1,26 +1,12 @@
...
+<% when :waiting %>
+  <%= render(Primer::OpenProject::FeedbackMessage.new(loading: true)) do |component|
+    component.with_heading(tag: :h2, id: 'waiting_title') { waiting_title }
+    component.with_description(id: 'waiting_subtitle') { waiting_subtitle }
+  end %>
+<% when :success %>
+  <%= render(Primer::OpenProject::FeedbackMessage.new) do |component|
+    component.with_heading(tag: :h2) { success_title }
+    component.with_description(id: 'waiting_subtitle') { success_subtitle }
+  end %>
```
Comment (akabiru) [inline modules/storages/app/components/storages/open_project_storage_modal_component/body.html.erb:11]: ✨

File: modules/storages/spec/requests/project_storages_open_spec.rb (hunk 1 above)
Comment (akabiru) [reply modules/storages/spec/requests/project_storages_open_spec.rb:78]: cc @ba1ash @apfohl  https://github.com/opf/openproject/pull/14386/files#r1423748760

**Review by akabiru — APPROVED** [review]
🚀

File: modules/storages/spec/requests/project_storages_open_spec.rb (hunk 1 above)
Comment (ba1ash) [reply modules/storages/spec/requests/project_storages_open_spec.rb:78]: I did that to make sure correct icon and text are delivered by turbo stream update.

**Pull request merged**